packages/mui/src/RadioWidget/RadioWidget.tsx

```diff
@@ -43,11 +45,16 @@ export default function RadioWidget<T = any, S extends StrictRJSFSchema = RJSFSc
   const row = options ? options.inline : false;
   const selectedIndex = enumOptionsIndexForValue<S>(value, enumOptions);
 
+  const { schemaUtils } = registry;
+  const displayLabel = schemaUtils.getDisplayLabel(schema, uiSchema);
+
   return (
     <>
-      <FormLabel required={required} htmlFor={id}>
-        {label || schema.title}
-      </FormLabel>
+      {displayLabel && (
+        <FormLabel required={required} htmlFor={id}>
+          {label || schema.title}
+        </FormLabel>
+      )}
```

Comment:
Can you make the same change in the `material-ui` theme as well. The code is nearly identical